guide/english/linux/basic-linux-commands/index.md

```diff
@@ -35,7 +35,8 @@ When starting out with linux, there are some basic commands everyone should know
 - Use the mv command to move files through the command line. We can also use the mv command to rename a file. For example, if we want to rename the file “text” to “new”, we can use 'mv text new'.
 
  10. **right-click** - copy and paste  
-- This one is less of a command and more of a how-to, however, it is very useful for doing almost anything in a terminal on linux. To begin, highlight text like normal and then "right-click" on your mouse to copy a selection. You should see the highlighted portion become un-highlighted, this means you copied the selection. Now "right-click" on where you want to paste the selection and you're done. 
+- This one is less of a command and more of a how-to, however, it is very useful for doing almost anything in a terminal on linux. To begin, highlight text like normal and then "right-click" on your mouse to copy a selection. You should see the highlighted portion become un-highlighted, this means you copied the selection. Now "right-click" on where you want to paste the selection and you're done.
+- You may also use shortcut keys for this. To begin, highlight the text like normal and then press `Ctrl` + `Shift` + `C`. Next, paste the copied text (wherever you like) by pressing `Ctrl` + `Shift` + `V`. 
```

Comment:
I don't think the `Shift` key is required in copy paste commands.

Reply:
While copying text from terminal, we use `Shift` key. You might want to refer to the link given here: https://help.gnome.org/users/gnome-terminal/stable/txt-copy-paste.html.en

Reply:
`Ctrl + Shift + V` doesn't work in the WSL terminal on Windows, while the already stated way of copy-pasting does work. It's best to not include commands specific to one environment in the general guide.